source/js.md

```diff
@@ -50,6 +50,43 @@ window.viewerReady = function (api, platform) {
 By default, customizations are applied to all device categories. This example shows how you could use the `platform` parameter to implement different customizations depending on the device the Viewer is running on.
 
 
+## Getting basic publication information
+
+``` javascript
+window.viewerReady = function (api, platform) {
+  var publication = api.publication;
+
+ /*
+  * Assuming your publication url is:
+  * https://view.publitas.com/my-group/my-publication/
+  */
+  publication.slug; // 'my-publication'
+  publication.groupSlug; // 'my-group'
+}
+```
+
+The `api.publication` property exposes information of the current publication. It contains the following properties:
+
+| Property      | Type        | Description         |
+| ------------- |-------------| ------------------- |
+| slug          | String      | publication identifier |
+| groupSlug     | String      | unique group identifier  |
+
+The `slug` and `groupSlug` properties are the building blocks of publication URLs, which have the structure
+
+*https://view.publitas.com/groupSlug/slug/`*
+
+They can be used to construct a unique identifier for the current publication (to be used for custom analytics tracking, for example).
```

Comment:
They can be used to construct a unique identifier for the current publication (e.g. to be used for custom analytics tracking).